src/panel/context/Timeline.tsx

```diff
@@ -96,22 +96,31 @@ const useTimelineDomain = () => {
       // Apply movement (limited left movement)
       domain.current = {
         ...domain.current,
-        start: max([newStart, startTime.current]) as number
+        start: newStart < startTime.current ? startTime.current : newStart
       };
     },
     [scale]
   );
 
   const handleZoom = useCallback((e: WheelEvent) => {
-    // Scale movement delta
+    e.preventDefault();
     const delta = e.deltaY * 0.01;
     const newZoom =
       e.deltaY < 0
         ? (max([0.2, domain.current.zoom + delta]) as number)
```

Comment:
Unrelated but lets get this done

```suggestion
        ? Math.max(0.2, domain.current.zoom + delta)
```